spec/menu_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'menu'
+
+describe Menu do
+  it 'creates an instance of Menu' do
+    expect(subject).to be_an_instance_of(Menu)   
+  end
+
+  it { is_expected.to respond_to(:view) }
```

Comment:
No need for these tests - you want to only test behaviour (the test below is good)